doc/source/how-to/manage-protobuf-definitions.rst

```diff
@@ -0,0 +1,163 @@
+Manage Protobuf definitions
+===========================
+
+Protobuf service definitions provide the API specification for underlying
+server implementations, so that each consuming client library has a clear
+contract for gRPC data messages. Ideally, the .proto files have a single
+repository established as the source of truth for the .proto files,
+organized by API version increment as the API definition expands and changes.
+Since most client libraries are custom implementations enhancing the developer
+experience when consuming the service, releasing the Protobuf definitions
+publicly gives full flexibility to the developer to operate at the abstraction
+layer they choose to do so.
+
+Maintain API definition repository
+----------------------------------
+
+The Protobuf definition of the service is language agnostic, so the repository
+containing the Protobuf files can be created within the top-level Ansys
```

Comment:
```suggestion
Because the Protobuf definition of the service is language agnostic, the repository
containing the Protobuf files can be created within the top-level Ansys
```